codecov.yml

```diff
@@ -2,32 +2,27 @@ coverage:
   status:
     project:
       default:
-        target: 80%
-        threshold: 50%
-      core:
-        flags: core
+        target: 70%
+
+      client:
+        flags: client
       react:
         flags: react
       playground:
         flags: playground
       react-playground:
         flags: react-playground
```

Comment:
is there a way to let codecov iterates `package` instead of defining them manually?